<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BuyWindow</class>
 <widget class="QWidget" name="BuyWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>5000</width>
    <height>5000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(190, 233, 116);</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QWidget" name="widget" native="true">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
border: 1px solid rgba(255, 255, 255, 140);
border-radius: 20px;
</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Совершение покупки</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="maximumSize">
             <size>
              <width>150</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Общий бюджет отдела:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="all_budget">
            <property name="styleSheet">
             <string notr="true">border-bottom-color: rgb(191, 233, 123);</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Наименование покупки:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="buy_name">
          <property name="styleSheet">
           <string notr="true">border-bottom-color: rgb(190, 233, 116);</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Описание:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="description">
          <property name="styleSheet">
           <string notr="true">border-color: rgb(190, 233, 116);</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Стоимость:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="sum">
          <property name="styleSheet">
           <string notr="true">border-bottom-color: rgb(191, 233, 123);</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Количество:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="count">
          <property name="styleSheet">
           <string notr="true">border-bottom-color: rgb(191, 233, 123);</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QLabel" name="label_7">
          <property name="maximumSize">
           <size>
            <width>90</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>MS Shell Dlg 2</family>
            <pointsize>8</pointsize>
            <weight>75</weight>
            <italic>false</italic>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">

</string>
          </property>
          <property name="text">
           <string>Общая цена:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="aii_sum">
          <property name="styleSheet">
           <string notr="true">border-bottom-color: rgb(191, 233, 123);</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="inform">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buy_window">
        <property name="cursor">
         <cursorShape>ArrowCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">


QPushButton {
color: black;
background-color: rgb(191, 233, 123);
border-radius: 10px;
width: 120px;
height: 30px;
} 
QPushButton:hover{
background-color: rgba(191, 233,140);
}

QPushButton:pressed{
background-color: rgba(191, 233,180);
}</string>
        </property>
        <property name="text">
         <string>Купить</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="Return_button">
          <property name="styleSheet">
           <string notr="true">


QPushButton {
color: black;
background-color: rgb(191, 233, 123);
border-radius: 10px;
width: 120px;
height: 30px;
} 
QPushButton:hover{
background-color: rgba(191, 233,140);
}

QPushButton:pressed{
background-color: rgba(191, 233,180);
}</string>
          </property>
          <property name="text">
           <string>Назад</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
